QUEST.tsv

```diff
@@ -1422,11 +1422,11 @@ QUEST_20150317_001420	Adaptation to Circumstances
 QUEST_20150317_001421	Last Mission of the Bishop
 QUEST_20150317_001422	Research Data
 QUEST_20150317_001423	Pharmacist's Favor
-QUEST_20150323_001424	Now go to Gintara Road if Sirdgela Forest. {nl}Spray this potion on the matsum dolls I placed and tell me the results. 
-QUEST_20150323_001425	Four scouts were sent to Sirdgela Forest not long ago but all of them went missing. {nl}Maybe they too the Thorn Forest too lightly.. Please find their whereabouts.
-QUEST_20150323_001426	I have no strength but I write this for someone who would pass by this area. {nl}Naktis.. It's a funny thing to wish for a curse from the Monarch of Curse. {nl}
+QUEST_20150323_001424	Now go to Gintara Road in Sirdgela Forest. {nl}Spray this potion on the matsum dolls I placed and tell me the results. 
+QUEST_20150323_001425	Four scouts were sent to Sirdgela Forest not long ago but all of them went missing. {nl}Maybe they went to the Thorn Forest without the proper equipment.. Please find their whereabouts.
```

Comment:
You could have switched it to "Maybe they took the Thorn Forest too lightly," it's what the Korean says.